pkg/edgeview/src/system.go

```diff
@@ -938,6 +967,213 @@ func dispAFile(f os.FileInfo) {
 	fmt.Printf("%s, %v, %d, %s\n", f.Mode().String(), f.ModTime(), f.Size(), f.Name())
 }
 
+func getTarFile(opt string) {
+	execCmd := strings.SplitN(opt, "tar/", 2)
+	if len(execCmd) != 2 {
+		fmt.Printf("tar needs a / and path input\n")
+		return
+	}
+	source := execCmd[1]
+	printColor(" - tar cmd: "+source, colorCYAN)
+	closePipe(true)
+
+	fi, err := os.Stat(source)
+	if err != nil {
+		fmt.Printf("find directory error: %v\n", err)
+		return
+	}
+
+	if !fi.IsDir() {
+		fmt.Printf("the path %s is not a directory\n", source)
+		return
+	}
+
+	ok := checkBlockedDirs(source)
```

Comment:
My comment probably belongs here and not in the cp command.

Reply:
If the user specifies tar of /persist, this chjeckedBlockDirs will return OK.
In that case, where do you check that /persist/clear or /persist/vault is not included?
I think the check here might be useful. since you can return an error. But you also need a checkBlockedDirs call inside the walk below.

Reply:
ok. One thing is '/persist' is exceeding 512Mbytes so it can not be tarred, but I'll put in explicit check.